### PR TITLE
Init CI

### PR DESCRIPTION
**Pull request by nikitabelonogov**

### CODE REVIEW

**Comment by esikachev** [comment]
Travis-ci is enabled now. You can retrigger jobs

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+python:
+  - "2.7"
+
+cache: pip
+
+install:
+  - pip install -r requirements.txt
+
+script:
```
Comment (zemuvier) [inline .travis.yml]: We don't need line with script for now (we don't have any script at all)

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    description='Simple chatbot',
+    long_description=long_description,
+
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
```
Comment (zemuvier) [inline setup.py]: Are we going to use python 3? :)

File: tests/tests.py [hunk 3]
```diff
@@ -0,0 +1 @@
+print('test')
```
Comment (zemuvier) [inline tests/tests.py]: New line in the end of each file is necessary

File: .travis.yml (hunk 1 above)
Comment (nikitabelonogov) [inline .travis.yml]: there is a blank test.py file

File: tests/tests.py (hunk 3 above)
Comment (nikitabelonogov) [inline tests/tests.py]: fixed

File: .travis.yml (hunk 1 above)
Comment (zemuvier) [inline .travis.yml]: Let's run tests via tox (https://tox.readthedocs.io/en/latest/)

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+language: python
+
+python:
+  - "2.7"
```
Comment (zemuvier) [inline .travis.yml]: Why do we need it? We use only python 2.7

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+cache: pip
```
Comment (zemuvier) [inline .travis.yml]: ditto

File: tests/tests.py (hunk 3 above)
Comment (zemuvier) [inline tests/tests.py]: I think this file is unnecessary

**Comment by zemuvier** [comment]
Fix conflicts and errors, and we will merge it

**Pull request closed without merging**